core/src/main/scala/knobs/Parser.scala

```diff
@@ -38,22 +38,22 @@ object ConfigParser {
   }
 
   /** Top-level parser of configuration files */
-  lazy val topLevel: Parser[List[Directive]] = "configuration" |: {
-    directives << skipLWS << realEOF
-  }
+  lazy val topLevel: Parser[List[Directive]] = topLevel(withImport = true)
```

Comment:
Renaming nullary `topLevel`, or adding a defaulted argument to it, would break both source and binary compatibility.  So do several of the other changes in this object.  Is there a way we can accomplish this without requiring a major version bump?